core/tauri/mobile/ios-api/Sources/Tauri/Invoke.swift

```diff
@@ -46,6 +46,9 @@ import UIKit
   }
 
   public func resolve(_ data: JsonObject) {
+    if (data.isEmpty) {
+      resolve()
```

Comment:
this is misleading as you are resolving an empty object but the value send through the bridge is actually `null`..

Reply:
Thanks for review. You are right. I will re-implement this so that it does not become a REJECT.

Reply:
Thanks for review. You are right.